tox.ini

```diff
@@ -33,6 +33,7 @@ commands = black --check {posargs:ingenialink tests}
 description = run type checks
 skip_install = true
 deps =
+    types-dataclasses==0.6.6
```

Comment:
Why did you add this? Ingenialink now works with python 3.9 or higher, and dataclasses are already included. It is because of rack?